src/docs/introduction.rst

```diff
@@ -1,12 +1,102 @@
+So what does that mean exactly? Did you ever wonder how the same login
+worked on all of the computers in the school lab? That’s because they
+are joined to a domain that’s part of an identity management solution.
+A centralized, redundant, and secure set of servers to manage users,
+groups, policies, and more.
 
-Maecenas fermentum purus sed lacinia vestibulum. Quisque justo orci, fermentum eu massa quis, efficitur dapibus erat. Morbi tristique sem id erat ultricies, ut eleifend orci consequat. Vestibulum urna orci, pretium ut lectus quis, elementum fermentum tellus. Praesent ligula sem, consequat et efficitur eu, laoreet quis quam. Suspendisse nec lorem diam. Integer et dolor ligula. Curabitur dapibus mi quis commodo viverra. Donec ut efficitur ipsum. Integer at bibendum leo, a volutpat dolor. Cras ut pulvinar leo, in euismod libero. Vivamus nec laoreet sem. Cras hendrerit est sit amet diam vehicula scelerisque.
+This isn’t necessary when you have a handful of machines to manage but
+when scaled up to tens, hundreds, and thousands something needs to be
+in place. You certainly can force the new guy to go to each machine to
+create a user, update their password, remove a user. However, that’s
+pretty mean and a waste of time when there is a much easier way of
+doing things.
+
+The community project FreeIPA *(Free Identity Policy and Auditing)* is
+an Identity Management solution. It’s also known as Red Hat IdM, which
+is the downstream name. FreeIPA, Red Hat IdM uses 389DS
+*(Directory Server)* for its database.
+
+These directory servers use the LDAP *(Lightweight Directory Access Protocol)*
+network protocol to representing objects to the computers on the network.
+Objects like a user, then that user object contains his user information
+like given name, surname, phone number, etc.
+
+*Please do not confuse LDAP with 389DS, 389DS is a directory server that
+uses the LDAP protocol on port 389.*
+
+Microsoft *(Active Directory Domain Services)* ADDS, but abbreviated to
+Microsoft AD. AD is another identity management solution for Windows. It has
+an LDAP server and shares many components with IPA. IPA and AD can have
+a trust created between the domain domains. Why? Consider if there was a
+company merger or acquisition. There won’t be a need to migrate users but
+instead, set up a trust and allow Bob from RedHat to administer
```

Comment:
I think I would replace RedHat and IBM with some more generic names. This example refers to RedHat acquisition by IBM which may not be obvious for everyone :)

Reply:
I know, the ...  indicates that it's a joke and the people who do get it, may laugh, but I'll generalize it.

Reply:
"There won’t be a need to migrate users but
instead, set up a trust and allow Bob from industry giant "SSSD Corp" to
administer newly acquired small business "LDAP LLC".

? =D